source/runbooks/revoke-network-access.html.md.erb

```diff
@@ -0,0 +1,117 @@
+
+This work is carried out from the core-vpc-(environment) environment e.g. core-vpc-production
+
+If you know which business unit the application under attack is in  you can disconnect the VPC from the internet by following these instructions.
+
+1. Go to VPC->Internet gateways
+2. Select the gateway (example is hmpps-preproduction-internet-gateway)
+3. Note the Internet gateway ID and the VPC ID
+4. Click on Actions, Detach from VPC
+5. If you are sure this is correct click on the Detach internet gateway button
+
+The selected internet gateway will be detached from the VPC.
+
+This change will affect all applications in the VPC that you have selected. No applications in the VPC will be accessible from the internet. 
+
+## Stopping access to the internet for applications on MP
+
+Due to the way Network Firewall applies rules in AWS (see [here](https://docs.aws.amazon.com/network-firewall/latest/developerguide/suricata-rule-evaluation-order.html) ), this section only covers changes to the VPC NACL. 
+
+This work is carried out from the core-vpc-(environment) environment e.g. core-vpc-production.
+
+The process needs to be done for all 3 NACLs listed. An example is
+
+- hmpps-production-data-nacl
+- hmpps-production-private-nacl
+- hmpps-production-public-nacl
+
+Follow the process below for each of these
+
+1. Go to VPC->Security->Network ACLs
+2. Click on the checkbox for the ACL you want to change then select **outbound rules**.
+3. Add a new Deny rule for 0.0.0.0/0, rule number 4999, which will prevent access to add the items listed after this point. All internet rules are 5000 and above.
+4. Save changes
+
+## Stopping access from the MOJ to an application via the Modernisation Platform Transit Gateway
+
+This work is carried out from the core-vpc-(environment) environment e.g. core-vpc-production
+
+As mentioned above the way checks are undertaken in AWS (see [here](https://docs.aws.amazon.com/network-firewall/latest/developerguide/suricata-rule-evaluation-order.html) ) mean adding a step for Deny may not work. **Note** any that are currently set to Deny in case this is manually changed back.
+
+Blocking the firewall is an option but this would mean that the access has already bypassed access into the system and blocking through the firewall may no be an effective solution. Blocking is done here to prevent unauthorised access. 
+
+The process needs to be done for all 3 NACLs listed. An example is
+
+- hmpps-production-data-nacl
+- hmpps-production-private-nacl
+- hmpps-production-public-nacl
+
+In this case the protected one needs to be amended as, although TCP references are in place, the IP addresses are the same as those listed for the other NACLs.
+
+You need to access this through the same route as above:
+
+1. Go to VPC->Security->Network ACLs
+2. Click on the checkbox for the one you want to change then select **inbound rules**.
+3. Amend all the rules on 4000, 4100 and 4200 so they are in a Deny state.
```

Comment:
This should say rules >= 4000 and < 5000 rather than list specific rules, as they can vary between each NACL and they can be expanded in the future. You could also mention at the beginning that we dedicate these rules to private CIDR ranges (hence only looking to amend the rules within this range).